vanguard_rpcxrpc_test.go

```diff
@@ -416,3 +419,150 @@ func TestMux_RPCxRPC(t *testing.T) {
 		})
 	}
 }
+
+func Test_grpcWebText(t *testing.T) {
```

Comment:
```suggestion
func TestTranscoder_GrpcWebText(t *testing.T) {
```

(Aside: at some point we should change the other battery of tests from `TestMux_...` to `TestTranscoder_...`, to reflect the actual API names.)